app/policies/external_activity_policy.rb

```diff
@@ -30,7 +30,7 @@ def set_private_before_matedit?
   end
 
   def copy?
-    user.present?
+    not_anonymous?
```

Comment:
Both no user and the anonymous is handled with the `not_anonymous?` policy helper so I think we are safe as is:

```  
def not_anonymous?
  user && !user.anonymous?
end
```